src/components/expenses/PersonSelect.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslation } from 'react-i18next'
+import { FormControl, MenuItem } from '@mui/material'
+import { useField } from 'formik'
+
+import { usePersonList } from 'common/hooks/person'
+import FormTextField from 'components/common/form/FormTextField'
+
+type Props = {
+  name: string
+}
+
+export default function PersonSelect({ name }: Props) {
```

Comment:
I think this might work as well

In any case the `name` prop can be optional when it has a default value

```suggestion
export default function PersonSelect({ name = 'personId' }) {
```